scripts/rebuild-packages.sh

```diff
@@ -14,13 +26,72 @@ if [ "$MODE" == "libs-only" ]; then
   glob_packages='!packages/*'
 fi
 
+function esbuild_cmd()
+{
+  format=$1
+
+  outdir="dist"
+  platform="browser"
+  if [ "$format" == "cjs" ]; then
+    outdir="dist-cjs"
+    platform="node"
+  fi
+
+  ./node_modules/.bin/esbuild \
+    --sourcemap \
+    --target=es2019 \
+    --bundle \
+    --platform=$platform \
+    --external:"bufferutil" \
+    --external:"utf-8-validate" \
+    --format=$format \
+    --outdir="./$outdir" \
+    --log-level=error \
+    src/index.ts
+}
+
+function fast_build()
+{
+  package=$1
+
+  echo "  ⏳@aragon/$package"
+
+  cd "packages/$package"
+
+  esbuild_cmd cjs & esbuild_cmd esm
+  wait
+
+  echo -e "\e[1A\e[K\u001b[32m  ✔️\u001b[0m @aragon/$package"
```

Comment:
This line prints a bit weird to me even though I have unicode characters.

Reply:
Could you post a screenshot? :thinking:

Also do you usually have emojis in your terminal?

Reply:
My bad I miss this message. Here it is:

<img width="751" alt="Screen Shot 2020-08-14 at 6 47 13 PM" src="https://user-images.githubusercontent.com/9082013/90295189-a2551b00-de5e-11ea-8add-454692887954.png">